core_backend/tests/rails/test_language_indentification.py

```diff
@@ -34,8 +34,13 @@ def test_language_identification(
 ) -> None:
     """Test language identification"""
     question = UserQueryRefined(query_text=content, query_text_original=content)
+    response = UserQueryResponse(
+        query_id=1,
+        content_response=None,
+        llm_response="Dummy response",
+        feedback_secret_key="feedback-string",
+    )
     if language not in available_languages:
         language = "UNKNOWN"
-    identified_language = identify_language(question).original_language
-    assert identified_language is not None
-    assert identified_language.value == language
+    _, response = _identify_language(question, response)
```

Comment:
Hmm llm is returning "YORUBA" instead of UNKNOWN and hence got this error:

<img width="744" alt="image" src="https://github.com/IDinsight/aaq-core/assets/7042047/0ad297a0-a2df-49d4-a2c7-487515c1636d">

Reply:
Thanks for picking this up. Fixed!